InferenceSystem/src/model_retrain_negative_samples.py

```diff
@@ -0,0 +1,46 @@
+
+import argparse
+import datetime
+import globals
+
+def main():
+    """
+    Example usage: 
+    python3 PrepareDataForPredictionExplorer.py --start_time "2020-07-25 19:15" --end_time "2020-07-25 20:15" --dataset_folder /Users/prakrutigogia/Documents/Microsoft/AlwaysBeLearning/MSHack/Round4
+    """
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--start_time', type=str, required=True, \
+        help='Start time in PST in following format 2020-07-25 19:15')
+    parser.add_argument('--end_time', type=str, required=True, \
+        help='End time in PST in following format 2020-07-25 20:15')
+    parser.add_argument('hydrophone', choices=list(globals.S3_STREAM_URLS.keys()), required=True, \
```

Comment:
Backslashes are not needed here. I'm wondering, is it some formatter that puts them here or were they put in manually? 🤔
If it's the latter, I suggest using Black formatter. For example this is how it formats these statements:
```py
    parser.add_argument(
        "hydrophone",
        choices=list(globals.S3_STREAM_URLS.keys()),
        required=True,
        help="Hydrophone location",
    )
```